fakes/v0.0/fakes-v0.0.graphql

```diff
@@ -0,0 +1,75 @@
+enum Type {
```

Comment:
Should the enum name be called `FakeType`. I would imagine there may be some conflicts with a generic name called `Type`

Reply:
Yea, probably, like `@listSize`... You can always rename the imports but better avoid the name clash in the first place. I'll prefix.